packages/material-ui/src/Step/Step.js

```diff
@@ -26,16 +26,21 @@ export const styles = {
 const Step = React.forwardRef(function Step(props, ref) {
   const {
     active = false,
+    // eslint-disable-next-line react/prop-types
```

Comment:
private props are rare enough so that we don't need to handle them. If missing validation becomes an issue when debugging we can revisit. But these components haven't been touched in a long time.